### ISSUE TITLE
System crashes on animals

### ISSUE THREAD

**Issue by smmaas** [TheGameCreators/GameGuruRepo#1791]
System crashes when user gets close to an animal

**Comment by UltraVox001** [comment]
I have no crashes with animals.
![animals](https://user-images.githubusercontent.com/96718175/149909877-8dcb1037-47c9-4a25-acd4-909dcb844c56.jpg)
Can you show me the visual of your map, with the animals ?

**Comment by UltraVox001** [comment]
It's strange. I don't have this problem : [video](https://i.gyazo.com/611ff8182b5fcefac4bc113e451f84d4.mp4)
The Original Behavior is : animals\mammal_generic.lua
Do you confirm that you have this Behavior ?

**Comment by smmaas** [comment]
yes mammal_generic. But the problem is maybe somewhere else. On an other level there is no crash. Is there no lua debugger?

**Comment by UltraVox001** [comment]
No, Max does not offer anything for the Lua. To my knowledge, Lua itself does not have a built-in governor, but many developers share their governor Lua code. You have to search on the web, or ask Lua users on the [GG Max Forum](https://forum.game-guru.com/board/38).

**Comment by smmaas** [comment]
I did some debugging and the system crashed in mammal_generic at line 133 with the command

RDFindPath(-15201.94140625,325.78454589844,36258.69921875,-14790.914314958,325.78454589844,36507.296914643)

I don't understand why you don't use error handling

**Comment by LeeBamberTGC** [comment]
In the latest internal build which works fine with mammals, line 133 is a END command. Impossible to crash there :)  See Friday's build.

**Comment by smmaas** [comment]
can be closed
